src/stream_base.h

```diff
@@ -160,7 +160,8 @@ class StreamBase : public StreamResource {
  public:
   enum Flags {
     kFlagNone = 0x0,
-    kFlagHasWritev = 0x1
+    kFlagHasWritev = 0x1,
+    kFlagNoShutdown = 0x2
```

Comment:
TTY seems to be in minority here, as other do support it. I guess I'd rather do it this way if this is not something serious for you?